app/serializers/activitypub/actor_serializer.rb

```diff
@@ -39,10 +39,12 @@ def shared_inbox
   delegate :moved?, to: :object
 
   def id
-    account_url(object)
+    object.instance_actor? ? instance_actor_url : account_url(object)
   end
 
   def type
+    return 'Application' if object.instance_actor?
```

Comment:
Feel this would be more readable in an if/elsif/else block.

Reply:
Alternatively, I considered using `object.actor_type` directly. I think this *should* be fine, but I'm not completely sure all the contexts it is used in. Will do an if/elsif else block for now.